File: gaia-sdk-javascript/src/graphql/request/type/AsyncSkillEvaluation.ts


import VariableRegistry from "../../../api/VariableRegistry"
import {Uuid, ISO8601, Struct} from "../../GaiaClient";
import {RuntimeState} from "../enumeration/RuntimeState";
import {SkillState} from "../enumeration/SkillState";
import {Order} from "../enumeration/Order";
import {OrderBy} from "../enumeration/OrderBy";

export class AsyncSkillEvaluation extends Array<(_:VariableRegistry) => string> {

    public tbd = () => { 
        this.push(_ => "tbd")
    };

    public render = (registry: VariableRegistry):String => this.map(e => e(registry)).join(" ");
}

File: gaia-sdk-python/graphql/request/type/AsyncSkillEvaluation.py


from typing import Callable
from api.VariableRegistry import VariableRegistry


class AsyncSkillEvaluation(list):

    def tbd(self):
        self.append(lambda x: "tbd")

    def render(self, registry: VariableRegistry):
        return " ".join(map(lambda e: e(registry), self))